Custom-Audio-Driver/External-Audio-Device/OTDefaultAudioDevice.m

```diff
@@ -47,6 +47,8 @@
 #endif
 
 static double kPreferredIOBufferDuration = 0.01;
+static double kToMicroSecond = 1000000;
```

Comment:
The ReadMe says it is build on top of the Basic App, but this is a custom audio driver code.

Reply:
This is  on a different sample. I will remove those changes